text_extensions_for_pandas/util.py

```diff
@@ -62,7 +62,8 @@ def pretty_print_html(column: Union["CharSpanArray", "TokenSpanArray"]) -> str:
             text_pieces.append("&#39;")
         elif text[i] == "$":
             # Dollar sign messes up Jupyter's JavaScript UI.
-            text_pieces.append("&#36;")
+            # Use backslash to avoid
```

Comment:
I think you meant to say "Embed a `<span>` element to prevent Jupyter from interpreting the dollar sign as the beginning of some embedded LaTex" in this comment; see issue #31 " here